Improved explenation around timing and added Graceful shutdown behaviour (#8114)
umbraco/UmbracoDocs: 17/umbraco-cms/extend-your-project/server-side-extensions/scheduling.md

`IDistributedBackgroundJob` is separate from `IRecurringBackgroundJob`, and is tracked in the database to ensure that only a single server runs the job at any given time. This also means that you are not guaranteed what server will run the job, but you are guaranteed that only one server will run it.

By default, each server polls for runnable jobs every 5 seconds, after an initial startup delay of 1 minute. The first execution of a given job happens at `startup + Delay + Period`, where `Period` is the job's own `Period` property. The polling interval and startup delay are both configurable in appsettings. See [Distributed jobs settings](../../develop-with-umbraco/configuration/distributedjobssettings.md) for the available options.

{% hint style="info" %}

> The job's `Period` (how often the job itself runs) and the `DistributedJobs:Period` setting (how often the host polls the database) are two different values that share the same name.

Reviewer: ```suggestion

The job's `Period` (run frequency) and the `DistributedJobs:Period` setting (database polling frequency) are two different values sharing the same name.

```
